### ISSUE TITLE
Project path tool-tip

### ISSUE THREAD

**Issue by msundquist** [oklai/koala#438]
Within the projects sidebar/pane, it would be convenient if there were a tool-tip or similar indication showing the full path of the project. Currently the only way to determine this is to (Right-Click -> Open Folder) on the project item.

Imagine a scenario where the user has dozens of project sub-directories added to the list of projects, all with the same directory structure (and thus the same label/title in the pane.) It's kind of difficult to determine which project to operate on.

**Comment by danielgetsthis** [comment]
Yes, I have this exact problem. I'm working on multiple projects that all have the same LESS folder name. Please see this screenshot. A small label with full path would do wonders. I would prefer a constant label as opposed to a tooltip that you would have to reveal with a mouse hover or click. Thank you!!
<img width="623" alt="screenshot 2017-05-19 09 57 56" src="https://cloud.githubusercontent.com/assets/4914026/26251127/36465fc0-3c7a-11e7-91e1-cde6343aab7a.png">
